app/models/early_career_payments/eligibility_matrix_calculator.rb

```diff
@@ -16,7 +9,7 @@ def initialize(eligibility)
     def eligible_later?
       return false if subject == :mathematics && itt_academic_year == "2018_2019"
 
-      MATRIX.dig(subject)&.include?(itt_academic_year) ? true : false
+      MATRIX.dig(subject.to_s, itt_academic_year).nil? ? false : true
```

Comment:
Didn't say anything before given current situation but glad you changed this 👍

Reply:
@nomasprime Yes I changed it as I changed the data structure to better reflect additional information. Its definitely better just using the `dig` method.